src/raftstore/store/worker/apply.rs

```diff
@@ -167,10 +167,33 @@ pub enum ExecResult {
     DeleteRange { ranges: Vec<Range> },
 }
 
+struct ApplyCallBack {
```

Comment:
Callback may be better?

Reply:
There is a `Callback` already.

Reply:
I mean ApplyCallback, you use the name Callback in other places.